source/scripting_v3/GTA.UI/Hud.cs

```diff
@@ -3,6 +3,7 @@
 // License: https://github.com/scripthookvdotnet/scripthookvdotnet#license
 //
 
+using System.Numerics;
```

Comment:
I think this using just specified wrong by accident, but we use `GTA.Math` for vector types, not `System.Numerics`.